Answers.md

```diff
@@ -1,6 +1,22 @@
 1.  Name 3 JavaScript Array/Object Methods that do not produce side-effects? Which method do we use to create a new object while extending the properties of another object?
+    - Array.object.map(), Array.object.filter(), Array.object.reduce(); Array.object.map() will create a new object while extending the properties of another object.
```

Comment:
The map method will actually not create a new object, it will create a new array. How do we copy an object and overwrite/add properties to it?

Reply:
with Array.object.split(), also the '...' operator